src/Services/DataService.test.ts

```diff
@@ -797,10 +954,55 @@ describe("DataService", () => {
       fakeService.deleteRecordEpic(ActionsObservable.of(deleteRecordAction), store)
         .subscribe(noop, noop,
           () => {
-            assert.isTrue(mockAdapter.deleteItem.calledOnceWith(expectedResult.id));
+            expect(mockAdapter.deleteItem.calledOnceWith(expectedResult.id)).to.be.true;
           });
     });
 
+    it("deleteItem should call normalize after serialize", () => {
```

Comment:
We can remove this test because `delete` doesn't run these methods.

Reply:
Oh, oops, no just rename this one, remove the next one.

Reply:
Ok